src/test/java/io/roastedroot/sqlite4j/ConnectionTest.java

```diff
@@ -463,10 +463,14 @@ public void failsOpenDatabaseNoCreateInConnectionString() throws Exception {
 
     @Test
     public void failsOpenDatabaseNoCreateUnavailablePath() throws Exception {
-        assertThatThrownBy(
-                        () ->
-                                DriverManager.getConnection(
-                                        String.format("jdbc:sqlite:/NON-EXISTENT9123/sample4.db")))
+        var path =
+                System.getProperty("os.name").toLowerCase().contains("win")
+                        // Use a network share that does not exist
+                        ? "\\\\NON-EXISTENT9123\\dir\\sample4.db"
+                        // this typically fails because you need root to create the dir
+                        : "/NON-EXISTENT9123/sample4.db";
```

Comment:
I think this test is interesting:  If you run the build as root on a posix system, I think this test will fail as the test will be able to create the /NON-EXISTENT9123 directory.